examples/experimental/ben/sklearn_experiments.ipynb

```diff
@@ -0,0 +1,966 @@
+{
```

Comment:
<p>Q: Add a <code>TODO</code>  for the user?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Experimental notebook</p>